doc/manual/manual.bib

```diff
@@ -611,6 +611,16 @@ @Article{HGSLM03
   pages =	 {15--30}
 }
 
+@Article{CSG12,
+  title={A comparison of numerical surface topography calculations in geodynamic modelling: An evaluation of the ‘sticky air’method},
+  author={Crameri, F and Schmeling, H and Golabek, GJ and Duretz, T and Orendt, R and Buiter, SJH and May, DA and Kaus, BJP and Gerya, TV and Tackley, PJ},
```

Comment:
We typically abbreviate first names with a trailing period (see the other entries) and spaces between multiple first names. Can you adjust?